config.go

```diff
@@ -200,8 +200,8 @@ type Config struct {
 	// BaseDir is a custom directory to store all aperture flies.
 	BaseDir string `long:"basedir" description:"Directory to place all of aperture's files in."`
 
-	// ProfilePort is the port on which the pprof profile will be served.
-	ProfilePort uint16 `long:"profile" description:"Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65535"`
+	// Profile is the port on which the pprof profile will be served.
+	Profile uint16 `long:"profile" description:"Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65535"`
```

Comment:
ah! interesting - TIL

Reply:
Yeah a weird quirk with the YAML parser, I think we might've ran into this w/ the sqlite stuff too, but alas the wisdom was forgotten....